### ISSUE TITLE
search records with _exists_ doesn't return records with empty but existing field

### ISSUE THREAD

**Issue by daviortega** [bitspill/oip#25]
The search:
https://api.oip.io/oip/o5/record/search?limit=12&page=1&q=_exists_:record.details.tmpl_7C6B7370

Does not return records like this one:
https://api.oip.io/oip/o5/record/get/ce5f4406fd867c5eff635c1b9dc43c8753940da16e31b91ddbc9ccbab68d69e2

But it should.

**Comment by bitspill** [comment]
The exists query returns fields with non-null values

https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-query-string-query.html#_field_names

**Comment by daviortega** [comment]
right, so why it isn't showing up? Is the response changing `null` to `{}`?

**Comment by bitspill** [comment]
`[]` and `{}` are both also considered non-existent/null for the purposes of ES queries

**Comment by daviortega** [comment]
any suggestion on how to find all records that has a specific template even if it is empty? (`null` for ES)?
